Database/Corrections/mopNPCFixes.lua

```diff
@@ -1814,6 +1818,11 @@ function MopNpcFixes.Load()
                 [zoneIDs.ISLE_OF_GIANTS] = {{67,75.2},{69.6,71.4},{69.8,71.6}}},
                 [zoneIDs.ISLE_OF_THUNDER] = {{50.4,42.8},{50.8,43.4},{51,43.6}},
         },
+        [70414] = { -- Skumblade Pillager
+            [npcKeys.name] = "Skumblade Pillager",
+            -- [npcKeys.spawns] = {[zoneIDs.VALE_OF_ETERNAL_BLOSSOMS] = {{}}}, Unknown location, not part of quest causing error
+            [npcKeys.zoneID] = zoneIDs.VALE_OF_ETERNAL_BLOSSOMS,
```

Comment:
wowhead says this NPC is added with 5.2.0 and is located in the "Troves of the Thunder King" Scenario 🤔 Just remove this correction and since the NPC is no longer listed in the objectives.

However I need to adjust the validations, to make sure the killCreditObjectives also don't have missing entries (like this one).